### PR TITLE
Desktop version, alignment fixs with media query, flexbox and grid

### PR DESCRIPTION
**Pull request by RafaelEchart**
Now compatible with all devices! From the start, this portfolio had an all-device approach, but in this commit, I improved every detail to make the experience better, hope you like it!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -175,8 +175,8 @@ <h1>Landing Page</h1>
     </section>
 
     <section class="division about-me">
```
Comment (Mihndim2020) [inline index.html:177]: - [ ] According to the design, the desktop version has a navigation bar with links various sections of page. The hamburger buttons is removed from the navigation bar in the desktop version. Please kindly fix that as it is part of the project requirements and best practices in the industry to stick to the clients design. See images below.

***Your implementation***
raph1

***The design***
raphdesign

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @RafaelEchart, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Correct use of media queries
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Site follows the design
:heavy_check_mark: Site is responsive

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION



# Status: Requires Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -230,7 +245,7 @@ <h4 class="subtitle-color">LET'S CONNECT</h4>
             checked="checked"
           />
           <h2 class="handle">
-            <label for="handle1">Languages <img src="./assets/arrow.png" class="arrow-margin" height="13" width="18" alt="arrow"></label>
+            <label for="handle1">Languages<img src="./assets/arrow.png" class="arrow-margin" height="13" width="18" alt="arrow"></label>
           </h2>
           <div class="content">
```
Comment (dasileker) [inline index.html:251]: - [ ] The language part you should have **JavaScript & HTML & CSS** and also they should be in inline position.

Capture web_23-9-2021_103651_www figma com

**Review by dasileker — CHANGES_REQUESTED** [review]

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @RafaelEchart,** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DanteAlonsoHT — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @RafaelEchart, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**